lib/auth-providers/cmd.js

```diff
@@ -33,10 +33,7 @@ module.exports = {
       const cmdEnv = config['cmd-env']
 
       const output = spawnSync(cmd, args, {
-        env: {
-          ...process.env,
-          ...cmdEnv
-        },
+        env: Object.assign(process.env, cmdEnv),
```

Comment:
Can you change this to:

```js
Object.assign({}, process.env, cmdEnv),
```

so we avoid mutating `process.env`?

Reply:
Sure. Done with it!